pkgs/development/interpreters/python/cpython/3.5/default.nix

```diff
@@ -67,6 +67,12 @@ in stdenv.mkDerivation {
   patches = [
     ./no-ldconfig.patch
     ./ld_library_path.patch
+  ] ++ optionals stdenv.isDarwin [
+    # Fix for https://bugs.python.org/issue24658
+    (fetchpatch {
+      url = "https://bugs.python.org/file45178/issue24658-3-3.6.diff";
```

Comment:
Is this url stable? I mean, is it likely that the author will update it?

Reply:
As far as I understand, “file45178” bit is unique id and file cannot be removed or modified. So yes.